utils/pron/pron.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <stdexcept>
+#include <string>
+#include <unordered_map>
+
+
+namespace utils
+{
+class Pron
+{
+ private:
+  Pron() = default;
```

Comment:
Instead of hiding the default ctor, better approach is to delete it:
```c++
class Pron
{
 public:
  Pron() = delete;
.
.
.
};
```

Reply:
I don't see the difference

Reply:
Yes you are right, please ignore this comment.